modules/concept-docs/pages/documents.adoc

```diff
@@ -30,115 +29,103 @@ Although query and path-based (Sub-Document) services are available, the simplic
 
-[source,python]
+[source,php,indent=0]
 ----
-collection.mutate_in("customer123", [SD.upsert("fax", "311-555-0151")])
+include::example$documents-concept.php[tag=mutate-in]
 ----
 
-or xref:6.0@server:n1ql:n1ql-language-reference/update.adoc[N1QL UPDATE] to update documents based on specific query criteria:
+or xref:7.0@server:n1ql:n1ql-language-reference/update.adoc[N1QL UPDATE] to update documents based on specific query criteria:
 
 [source,sql]
 ----
-update `default` SET sale_price = msrp * 0.75 WHERE msrp < 19.95;
+update `travel-sample`.inventory.airline SET sale_price = msrp * 0.75 WHERE msrp < 19.95;
 ----
 ====
 
-include::6.5@sdk:shared:partial$documents.adoc[tag=get_generic]
+include::7.0@sdk:shared:partial$documents.adoc[tag=get_generic]
 
 [source,sql]
 ----
-SELECT * FROM default USE KEYS ["docid"];
+SELECT * FROM `travel-sample`.inventory.airport USE KEYS ["airport_1254"];
 ----
 
 or
 
 [source,sql]
 ----
-SELECT * FROM default WHERE META(default).id = "docid";
+SELECT * FROM `travel-sample`.inventory.airport WHERE META().id = "airport_1254";
 ----
 
 You can also retrieve _parts_ of documents using xref:subdocument-operations.adoc[sub-document operations], by specifying one or more sections of the document to be retrieved
 
-[source,python]
+[source,php,indent=0]
 ----
-name, email = cb.retrieve_in('user:kingarthur', 'contact.name', 'contact.email')
+include::example$documents-concept.php[tag=lookup-in]
 ----
 
 // Counters
 
-include::6.5@sdk:shared:partial$documents.adoc[tag=counters1]
+include::7.0@sdk:shared:partial$documents.adoc[tag=counters1]
 
-[source,java]
+[source,php,indent=0]
 ----
-//  java example:
-String counterDocId = "counter-doc";
-// Increment by 1, creating doc if needed
-collection.binary().increment(counterDocId);
-// Decrement by 1
-collection.binary().decrement(counterDocId);
-// Decrement by 5
-collection.binary().decrement(counterDocId,
-DecrementOptions.decrementOptions().delta(5));
+include::example$documents-concept.php[tag=counters]
 ----
 
-include::6.5@sdk:shared:partial$documents.adoc[tag=counters2]
+include::7.0@sdk:shared:partial$documents.adoc[tag=counters2]
 
-[source,python]
+[source,php,indent=0]
 ----
-# Python example:
-rv = cb.get('counter_id')
-value, cas = rv.value, rv.cas
-if should_increment_value(value):
-  cb.upsert('counter_id', value + increment_amount, cas=cas)
+include::example$documents-concept.php[tag=counter-increment]
 ----
 
-include::6.5@sdk:shared:partial$documents.adoc[tag=counters3]
+include::7.0@sdk:shared:partial$documents.adoc[tag=counters3]
 
 
 === Use Cases
 
-The SDK provides a high-level abstraction over the simple `incr()`/`decr()` of Couchbase Server's memcached binary protocol, using `collections.binary()`. 
+The SDK provides a high-level abstraction over the simple `incr()`/`decr()` of Couchbase Server's memcached binary protocol, using `Collection->binary()`.
```

Comment:
good catch, there's quite a few of these spotted around, where I'm not sure if the methods referred to are the *server* names, or a template from another SDK, etc.

(For example, I'm assuming `incr()`/`decr()` are the server protocol so unchanged?)

Reply:
yeah exactly, It was confusing - I would say if you see it change it, but if pressed for time we can review these later.